src/Components/Notifications/NotificationsList.tsx

```diff
@@ -350,7 +350,26 @@ export default function NotificationsList({
     })
       .then((res) => {
         if (res && res.data) {
-          setData(res.data.results);
+          const toChangeId = [
+            "PATIENT_CONSULTATION_UPDATED",
+            "PATIENT_CONSULTATION_UPDATE_CREATED",
+            "PATIENT_CONSULTATION_UPDATE_UPDATED",
+            "PATIENT_CONSULTATION_CREATED",
+          ];
+          const modifiedData = res.data.results.map((notification: any) => {
+            if (toChangeId.includes(notification.event)) {
+              return {
+                ...notification,
+                message: notification.message.replace(
+                  "Consultation",
+                  "Log Updates",
+                ),
+              };
+            } else {
+              return notification;
+            }
```

Comment:
This doesn't seem to be the right way to do it.

Why is patient consultation created event being renamed to log update? Consultation creation is not relevant to log updates right?

Reply:
Can you clarify if the way I am modifying data is wrong or is it just the part that I have 'PATIENT_CONSULTATION_CREATED' in toChangeId wrong?

Reply:
Consultation Create notification message need not be replaced with log updates right?